Gosu/Window.h

```diff
@@ -28,7 +28,7 @@ int Gosu_Window_width(Gosu_Window* window);
 int Gosu_Window_height(Gosu_Window* window);
 bool Gosu_Window_fullscreen(Gosu_Window* window);
 double Gosu_Window_update_interval(Gosu_Window* window);
-int Gosu_Window_resizable(Gosu_Window* window);
+bool Gosu_Window_resizable(Gosu_Window* window);
```

Comment:
This is probably a matter of taste, but I think is_resizable would look better in C. C++ often uses methods like `empty()` instead of `is_empty()`, but I'm not sure if this pattern applies to C, where names are more random. `isupper` and friends have an `is` prefix, for what it's worth.